website/content/docs/nia/api/status.mdx

```diff
@@ -225,3 +226,78 @@ Response:
   }
 }
 ```
+
+## Cluster status
+
+The `/v1/status/cluster` API endpoint returns information about high-availability clusters and its members, such as health status and leadership. This endpoint is only supported when using CTS in [high availability mode](/docs/nia/usage/run-ha). If you call the endpoint without configuring CTS for high availability, then CTS prints an error to the console. Refer to [Error Messages](/docs/nia/usage/errors-ref) for information about CTS error messages. 
+
+| Method | Path              | Response format    |
+| ------ | ----------------- | ------------------ |
+| `GET`  | `/status/cluster` | `application/json` |
+
+### Request parameters
+
+Currently no request parameters are offered for the overall status API.
+
+### Request statuses
+
+- `200`: Successfully retrieved the cluster status
+
+### Response fields
+
+The following table describes the responses that the `status/cluster` endpoint can send. 
+
+| Field | Type | Description |
+| --- | ---- | --- |
+| `cluster_name` | string | Identifies the name of the cluster. |
+| `members` | array | Contains an array of [member objects](#member-objects). Each object in the `members` array represents a CTS instance. |
+
+#### Member objects
+
+The following table describes the fields available for objects in the `members` array.  
+
+| Field | Type | Description |
+| --- | ---- | --- |
+| `address` | string | Indicates the location of the instance. The address is only included in the response if the `high_availability.instance.address` option is configured on the leader instance. Refer to the [high availability instance configuration](/docs/nia/configuration#high-availability-instance) reference for additional information. |
+| `healthy` | boolean | Indicates the health of the service instance health. Refer to [Service Registration](/docs/nia/configuration#service-registration) for additional information. |
+| `id` | string | Indicates the service registration ID. Refer to [Service Registration](/docs/nia/configuration#service-registration) for additional information. |
+| `leader` | boolean | Identifies the cluster leader. |
+| `service_name` | string | Identifies the name of the service that the instance represents. Refer to [Service Registration](/docs/nia/configuration#service-registration) for additional information. Default is `consul-terraform-sync`. |
```

Comment:
I would suggest saying that the value is set by `service_name` in `Service Registration.` It already contains the default, and it is where the default is set.

That way if the default changes in the future, you don't have to change that documentation in two places

Reply:
Also I think because this is a returned object, and not configuration, the default isn't necessarily critical information